packages/inter-protocol/src/proposals/replaceElectorate.js

```diff
@@ -0,0 +1,229 @@
+  const highPrioritySendersManager = await highPrioritySendersManagerP;
+
+  if (!highPrioritySendersManager) {
+    throw Error('highPrioritySendersManager is not defined');
+  }
+
+  const { addressesToAdd, addressesToRemove } = highPrioritySendersConfig;
+
+  for (const addr of addressesToAdd) {
+    trace(`Adding ${addr} to High Priority Senders list`);
+    await E(highPrioritySendersManager).add(
+      HIGH_PRIORITY_SENDERS_NAMESPACE,
+      addr,
+    );
+  }
+
+  for (const addr of addressesToRemove) {
+    trace(`Removing ${addr} from High Priority Senders list`);
+    await E(highPrioritySendersManager).remove(
+      HIGH_PRIORITY_SENDERS_NAMESPACE,
+      addr,
+    );
+  }
+};
+
+const inviteECMembers = async (
+  { consume: { namesByAddressAdmin } },
+  { options: { voterAddresses = {}, economicCommitteeCreatorFacet } },
+) => {
+  trace('Create invitations for new committee');
+
+  const invitations = await E(
+    economicCommitteeCreatorFacet,
+  ).getVoterInvitations();
+  assert.equal(invitations.length, values(voterAddresses).length);
+
+  trace('Distribute invitations');
+  /** @param {[string, Promise<Invitation>][]} addrInvitations */
+  const distributeInvitations = async addrInvitations => {
+    await Promise.all(
+      addrInvitations.map(async ([addr, invitationP]) => {
+        const [voterInvitation] = await Promise.all([invitationP]);
+        trace('Sending voting invitations to', addr);
+        await reserveThenDeposit(
+          `econ committee member ${addr}`,
+          namesByAddressAdmin,
+          addr,
+          [voterInvitation],
+        );
+      }),
+    );
+  };
+
+  await distributeInvitations(zip(values(voterAddresses), invitations));
+};
+
+const startNewEconomicCommittee = async (
+  {
+    consume: { board, chainStorage, diagnostics, zoe },
+    produce: { economicCommitteeKit, economicCommitteeCreatorFacet },
+    installation: {
+      consume: { committee },
+    },
+    instance: {
+      produce: { economicCommittee },
+    },
+  },
+  { options: { committeeName, committeeSize } },
+) => {
+  const COMMITTEES_ROOT = 'committees';
+
+  trace('startNewEconomicCommittee');
+
+  trace(`committeeName ${committeeName}`);
+  trace(`committeeSize ${committeeSize}`);
+
+  const committeesNode = await makeStorageNodeChild(
+    chainStorage,
+    COMMITTEES_ROOT,
+  );
+  const storageNode = await E(committeesNode).makeChildNode(
+    sanitizePathSegment(committeeName),
+  );
+
+  const marshaller = await E(board).getPublishingMarshaller();
+
+  trace('Starting new EC Committee Instance');
+
+  const privateArgs = {
+    storageNode,
+    marshaller,
+  };
+
+  const startResult = await E(zoe).startInstance(
+    committee,
+    {},
+    { committeeName, committeeSize },
+    privateArgs,
+    'economicCommittee',
+  );
+  const { instance, creatorFacet } = startResult;
+
+  trace('Started new EC Committee Instance Successfully');
+
+  economicCommitteeKit.resolve(
```

Comment:
usually needs `economicCommitteeKit.reset()` first. If it's already set and `reset()` isn't called, it fails silently. 😞